drafts/w3f-#5475-coretime-marketplace-draft.md

```diff
@@ -18,23 +18,129 @@ To fully follow the material on this page, it is recommended to be familiar with
+
+At the normal Bulk Coretime Marketplace the price depends on Demand.
+
+Each month.<br />
+Offered Cores (max to be sold).<br />
+Ideal (Best if sold).<br />
+If Sold > Ideal in current month, the price raises 5% for the next selling.
+
+
+**Bulk Sales** <br />
+- A sale of Bulk Coretime occurs on the Coretime-chain every `BULK_PERIOD` blocks.
+- In every sale, a `BULK_LIMIT` of individual Regions are offered for sale.
+- Each Region offered for sale has a different Core Index, ensuring that they each represent an independently allocatable resource on the Polkadot UC.
+- The Regions offered for sale have the same span: they last exactly `BULK_PERIOD` blocks, and begin immediately following the span of the previous Sale's Regions. - The Regions offered for sale also have the complete, non-interlaced, Core Mask.
+- The Sale Period ends immediately as soon as span of the Coretime Regions that are being sold begins. At this point, the next Sale Price is set according to the previous Sale Price together with the number of Regions sold compared to the desired and maximum amount of Regions to be sold. See Price Setting for additional detail on this point.
+- Following the end of the previous Sale Period, there is an Interlude Period lasting `INTERLUDE_PERIOD` of blocks. After this period is elapsed, regular purchasing begins with the Purchasing Period.
+- This is designed to give at least two weeks worth of time for the purchased regions to be partitioned, interlaced, traded and allocated.<br />
+
+**Region**<br />
+A Core bought by Bulk is made out of parts  Regions (Bulk Coretime Assets)
+The Owner of the Core can
+Assign a Task (like Parachain) to the Region
+Transfer some Regions to another Owner
+Put Regions on Instantaneous Coretime Marketplace (like Lastic or Cortego)
+Split Regions into smaller pieces
+***
 ### Instantaneous Coretime
 
 <!-- Breaking down the details of the concepts -->
+#### Instantaneous Coretime and Spot Price Multiplier
+
+Instantaneous Coretime, previously referred to as "Parathreads" or "Pay-as-you-go Parachains," is sold on the Relay Chain immediately prior to its usage on a block-by-block basis. It allows for the immediate allocation of core resources for tasks without the need to pre-schedule or participate in periodic Bulk Coretime sales. Instantaneous Coretime credits are non-transferable and non-refundable, only usable for immediate core allocation.
+
+#### Instantaneous Coretime Marketplace
+
+On the Instantaneous Coretime Marketplace, the price will also depend on demand (% of OrderBook filled).
+
+#### Spot Price Multiplier
+
+##### Parameters:
 
```

Comment:
We don't want to include technical code in the Polkadot Wiki. The wiki supports audiences which are researchers as well, not only programmers.